pkg/kubernetes/difftofile.go

```diff
@@ -0,0 +1,35 @@
+package kubernetes
+
+import (
+	"github.com/pkg/errors"
+	"os"
+)
+
+// always create file for consistency
```

Comment:
This could probably be cleaned up and expanded a bit to form a proper sentence.
Also the function name seems weird since it has nothing to do with diff. Maybe
`writeStringToFile` or something like that?